librz/include/rz_arch.h

```diff
@@ -1,65 +1,16 @@
-// SPDX-FileCopyrightText: 2021 RizinOrg <info@rizin.re>
+// SPDX-FileCopyrightText: 2022 RizinOrg <info@rizin.re>
```

Comment:
shouldn't this file be deleted completely?

Reply:
no, is there for the future.

Reply:
removed.